zhaquirks/danfoss/thermostat.py

```diff
@@ -36,6 +44,17 @@ class DanfossThermostatCluster(CustomCluster, Thermostat):
     }
 
     manufacturer_attributes = {
+        0x0000: ("local_temperature", t.int16s),
+        0x0003: ("abs_min_heat_setpoint_limit", t.int16s),
+        0x0004: ("abs_max_heat_setpoint_limit", t.int16s),
+        0x0008: ("pi_heating_demand", t.int8s),
+        0x0012: ("occupied_heating_setpoint", t.int16s),
+        0x0015: ("min_heat_setpoint_limit", t.int16s),
+        0x0016: ("max_heat_setpoint_limit", t.int16s),
+        0x001B: ("control_sequence_of_operation", t.enum8),
+        0x001C: ("system_mode", t.enum8),
+        0x0025: ("thermostat_programming_operation_mode", t.bitmap8),
+        0x0030: ("setpoint_change_source", t.enum8),
```

Comment:
These are all standard attributes on the thermostat cluster. Is there a reason you added them to the `manufacturer_attributes` collection?

Reply:
No sorry, his is an excess of enthousiasm while reading the Danfoss reference, which does not separate those types of attributes.
I am going to modify the pull request, are your three comments exhaustive about the overlap of attributes or should I check them all ?

Reply:
Double check them and I’ll peek again when you’re done